beta/src/pages/learn/thinking-in-react.md

```diff
@@ -25,45 +26,47 @@ The JSON API returns some data that looks like this:
 ]
 ```
 
-The mockup looks like this:
+मॉकअप इस तरह दिखता है:
 
 <img src="/images/docs/s_thinking-in-react_ui.png" width="300" style={{margin: '0 auto'}} />
 
-To implement a UI in React, you will usually follow the same five steps.
+React  में  UI  को लागू करने के लिए, आप आमतौर पर एक ही पांच चरणों का पालन करेंगे।
+
+## Step 1: एक कुम्पोनेंट पदानुक्रम में UI को तोड़ो {/*step-1-break-the-ui-into-a-कौम्पोनॅन्ट-hierarchy*/}
 
-## Step 1: Break the UI into a component hierarchy {/*step-1-break-the-ui-into-a-component-hierarchy*/}
+मॉकअप में प्रत्येक कॉम्पोनेन्ट और सब कंपोनेंट के चारों ओर बॉक्स बनाकर शुरू करें और उनका नामकरण करें।
```

Comment:
`कॉम्पोनेन्ट` => `कौम्पोनॅन्ट`

update reference to all components work to `कौम्पोनॅन्ट` we want to keep it consistent